api/internal/handler/data_loader/route_export_test.go

```diff
@@ -1903,10 +1904,66 @@ func TestExportRoutesSameURI(t *testing.T) {
 	assert.True(t, getCalled)
 }
 
+func TestExportRoutesParameterEmpty(t *testing.T) {
+	// Error test when pass parameter is null
+	r1 := `{
+		"uris": ["/test-test"],
+		"name": "route_all",
+		"desc": "所有",
+		"methods": ["GET"],
+		"hosts": ["test.com"],
+		"status": 1,
+		"upstream": {
+			"nodes": {
+				"172.16.238.20:1980": 1
+			},
+			"type": "roundrobin"
+		}
+}`
+	var route *entity.Route
+	err := json.Unmarshal([]byte(r1), &route)
+	assert.Nil(t, err)
+	mStore := &store.MockInterface{}
+	mStore.On("Get", mock.Anything).Run(func(args mock.Arguments) {
+	}).Return(route, nil)
+
+	h := Handler{routeStore: mStore}
+	exportInput := &ExportInput{}
+	exportInput.IDs = ""
+	ctx := droplet.NewContext()
+	ctx.SetInput(exportInput)
+	_, err1 := h.ExportRoutes(ctx)
+	assert.Equal(t, errors.New("Parameter IDs cannot be empty"), err1)
```

Comment:
What about pre defining error object and reusing them here, if the description changed, we have to change it here.

```go
var ErrEmptyID = errors.New("empty id")
```

Reply:
I personally think that this return error is used by this test alone and only once. I don't think it's necessary to declare a new variable